UNITY-9319 REST API - Update Product POST Endpoint - productNumber no… (#1337)
ePages-de/epages-docs: apps/data-types.md

| Attribute | Type | Description |
| - | - |  - |

> | productNumber | string | The product number (if not passed through, an automatically system generated product number will be used). |

Reviewer: ```suggestion
| productNumber | string | The product number. If no product number is provided, an automatically system generated one will be used). |
```
Could we rephrase it like that? The part in the brackets should be an individual sentence, in my opinion.

Author: If you want :)
